### ISSUE TITLE
[Project Repo] project structure, branching, issues and PRs for devs

### ISSUE THREAD

**Issue by akhmpr** [nic-delhi/AarogyaSetu_Android#201]
`Hello Devs and code owners, I would like to propose below or something similar or better, in order to have organized repository and better code control.`

![AS](https://user-images.githubusercontent.com/21245817/83019436-f20d0e80-9ff4-11ea-82ea-4ba15a6c6e12.png)
 
`let me know your thoughts.` :bowtie:

**Comment by Schidster** [comment]
@akhmpr , the strategy you have proposed (git flow) is kinda overkill for a rather small project like this. I also see many open pull requests and issues which indicates that the devs aren't active enough.
I would rather suggest a fast paced development cycle called [github-flow](https://guides.github.com/introduction/flow/) which probably everyone knows as its far more simple and fast paced.

**Comment by akhmpr** [comment]
> @akhmpr , the strategy you have proposed (git flow) is kinda overkill for a rather small project like this. I also see many open pull requests and issues which indicates that the devs aren't active enough.
> I would rather suggest a fast paced development cycle called [github-flow](https://guides.github.com/introduction/flow/) which probably everyone knows as its far more simple and fast paced.

yeah right, it might be overkill 👍 considering the activity level.

**Comment by urmilshroff** [comment]
> @akhmpr , the strategy you have proposed (git flow) is kinda overkill for a rather small project like this. I also see many open pull requests and issues which indicates that the devs aren't active enough.

I am a little surprised to see that an app of this scale and impact was open sourced without having any proper contribution guidelines, issue/PR templates or development flow in place. It seems as if they did it as an afterthought.

Heck, there's no organization description or photo either, which could make people question the legitimacy of this being the official project (I know it is, but still).

**Comment by akhmpr** [comment]
> > @akhmpr , the strategy you have proposed (git flow) is kinda overkill for a rather small project like this. I also see many open pull requests and issues which indicates that the devs aren't active enough.
> 
> I am a little surprised to see that an app of this scale and impact was open sourced without having any proper contribution guidelines, issue/PR templates or development flow in place. It seems as if they did it as an afterthought.
> 
> Heck, there's no organization description or photo either, which could make people question the legitimacy of this being the official project (I know it is, but still).

I doubt they owned a github organization or account before, so I assume pretty new to this. Now that repo is public and being flooded by PR and Issues, they will probably take much time to review and actually implement any proposed PoC changes. Only doc update changes made it to master till now i guess.
